sdk/src/types/block/output/delegation.rs

```diff
@@ -236,21 +239,22 @@ impl From<&DelegationOutput> for DelegationOutputBuilder {
     }
 }
 
-/// Describes a Delegation output, which delegates its contained IOTA tokens as voting power to a validator.
+/// An output which delegates its contained IOTA tokens as voting power to a validator.
 #[derive(Clone, Debug, Eq, PartialEq, Ord, PartialOrd, Hash)]
 pub struct DelegationOutput {
-    // Amount of IOTA tokens held by the output.
+    /// Amount of IOTA tokens to deposit with this output.
     amount: u64,
-    /// The amount of delegated coins.
+    /// Amount of delegated IOTA tokens.
     delegated_amount: u64,
-    /// Unique identifier of the Delegation Output, which is the BLAKE2b-256 hash of the Output ID that created it.
+    /// Unique identifier of the delegation output.
```

Comment:
learning question: is the explanation not correct anymore?

Reply:
Just unnecessary here, it's already in the type definition. No need to put it every time we have a delegation ID field.